cea/optimization/preprocessing/decentralized_buildings.py

```diff
@@ -246,9 +253,9 @@ def calc_new_load(mdot, TsupDH, Tret, gv):
         # Check the GHP area constraint
         for i in range(10):
             QGHP = (1-i/10) * Qnom
-            areaAvail = geothermal_potential.ix[building_name,"Area_geo"]
+            geothermal_potential = geothermal_potential_data.set_index('Name')
```

Comment:
great work at fixing this. However, better do this line out of the loop. It is necessary only once.

Reply:
Aaah..i am an idiot. Fixed it in the new commit. Thanks